app/decorators/user_decorator.rb

```diff
@@ -154,6 +165,14 @@ def phone_number
     @user.phone_number
   end
 
+  def is_over_18
+    @user.is_over_18 ||= ''
+  end
+
+  def consented_to_background_check
+    @user.consented_to_background_check ||= ''
```

Comment:
If all users need to consent to a background check, we should probably enforce this on the db/model level so that there will never be a nil value here.